packages/jellyfish-testing/__tests__/testingcontainer.test.ts

```diff
@@ -0,0 +1,136 @@
+import { Testing } from '@defichain/jellyfish-testing'
+import { MainNetContainer, RegTestContainer, StartFlags, TestNetContainer } from '@defichain/testcontainers'
+import { TestingWrapper } from '../src/testingwrapper'
+
+const testingWrapper = new TestingWrapper()
+describe('create a single test container using testwrapper', () => {
+  let testing: Testing
+
+  afterEach(async () => {
+    await testing.container.stop()
+  })
+
+  it('should be able to create and call single regtest masternoderegtest container', async () => {
+    testing = testingWrapper.create()
+    await testing.container.start()
+
+    // call rpc
+    let blockHeight = await testing.rpc.blockchain.getBlockCount()
+    expect(blockHeight).toStrictEqual(0)
+    await testing.generate(1)
+    blockHeight = await testing.rpc.blockchain.getBlockCount()
+    expect(blockHeight).toStrictEqual(1)
+  })
+
+  it('should be able to create regtest non masternode container', async () => {
+    testing = testingWrapper.create(1, index => new RegTestContainer()) as Testing
```

Comment:
I think we can potentially remove `index` in the methods here as they are unused?

Reply:
yup you are right